test/ts/index.tsx

```diff
@@ -1,38 +1,49 @@
-import { h, app, ActionsType, View } from "hyperapp"
+import { h, app, View, StateType, UnwiredActions, UnwiredAction } from 'hyperapp'
 
 namespace Counter {
-  export interface State {
+  export interface State extends StateType {
     count: number
+    clickCount: number
   }
 
-  export interface Actions {
-    down(): State
-    up(value: number): State
+  export interface Actions extends UnwiredActions<State, Actions> {
+    down: UnwiredAction<State, Actions>
```

Comment:
UnwiredActions is rather long. What's wrong with ActionsType or ActionsTemplate?

Reply:
Nothing, just wanted to put something that's not even remotely confusing

Reply:
I'm for changing UnwiredAction to ActionsTemplate. But what about WiredActions? It's an internal type that no-one should be using anyway.

Reply:
I see. Don't change it yet please.